solvers/cd.py

```diff
@@ -77,7 +77,6 @@ def run(self, n_iter):
             self.w = self.cd(self.X, self.y, self.l1_ratio,
                              self.lmbda, L, n_iter)
 
-    @staticmethod
```

Comment:
no

Reply:
I auto-opened PRs to fix ``get_next``, so there might be some misses.

could you _"suggest changes / I approve them"_ to get things quickly done?